bundles/org.openhab.binding.mqtt.espmilighthub/README.md

```diff
@@ -152,6 +154,21 @@ For example:
 | 0xB4CA    | 8        | 0xB4CA8  |
 | 0xB4CA    | 0        | 0xB4CA0  |
 
+## Using the group 0
+
+The group 0 (or ALL group) with the Group ID 0 can be used to control all bulbs that are paired with one specific remote at once.
+While this functionality can also be achieved by using openHAB groups with even greater flexibility the group 0 must be setup if you want to capture remote events for the ALL group to keep physical devices synchronized to their openHAB representations.
+Milight remotes send all commands with the Group ID 0 after the master ON/OFF buttons have been used.
+If the group 0 has not been setup these events will be lost and your Item states will no longer be synchonized with the actual device states until you issue a command via openHAB.
+If you do not use a remote at all or you only control other bulbs than the ones controlled by openHAB you should not need to setup the ALL group.
+
+Since the group 0 is not needed in every case the autodiscovery feature will not detect this group as a Thing automatically.
+To create the group manually proceed as described in section [Important for Textual Configuration](#important-for-textual-configuration) or use the openHAB 3 MainUI for this purpose.
```

Comment:
Do not mention a version of openHAB, as when V4 is released this becomes outdated and hard to know it needs updating.

```suggestion
To create the group, use textual files or the openHAB UI to manually add a Thing with the correct Unique ID as described in section [Important for Textual Configuration](#important-for-textual-configuration).
```